FirebaseRemoteConfigSwift/Sources/FirebaseRemoteConfigValueDecoderHelper.swift

```diff
@@ -47,4 +52,18 @@ struct FirebaseRemoteConfigValueDecoderHelper: FirebaseRemoteConfigValueDecoding
     }
     return value
   }
+
+  func arrayValue() throws -> [AnyHashable] {
+    guard let value = value.jsonValue as? [AnyHashable] else {
+      throw RemoteConfigValueDecoderError.arrayDecodeFailure
+    }
+    return value
+  }
+
+  func dictionaryValue() throws -> [String: AnyHashable] {
```

Comment:
Are just constrained to `String` keys? Could the plist dictionary have integer keys that are decoded as such?

Reply:
I believe the keys must be `String`. See [FirebaseDataEncoder.swift](https://github.com/firebase/firebase-ios-sdk/pull/10046/files#diff-db170d9b3f2582a4ba0a083c21fddd7da24eca78abaa33a0f330a3df8184eb19) line 2479 below.